src/mlpack/methods/lsh/lsh_search.hpp

```diff
@@ -143,7 +159,8 @@ class LSHSearch
               const size_t k,
               arma::Mat<size_t>& resultingNeighbors,
               arma::mat& distances,
-              const size_t numTablesToSearch = 0);
+              const size_t numTablesToSearch = 0,
+              size_t T = 0);
```

Comment:
Can you document what the `T` parameter is here?  We should probably mark it const too if we can.